src/vaccel.c

```diff
@@ -60,16 +60,23 @@ static int load_backend_plugin(const char *path)
 
 static int load_backend_plugins(char *plugins)
 {
-	char *plugin = strtok(plugins, ";");
-	while (plugin) {
-		int ret = load_backend_plugin(plugin);
-		if (ret != VACCEL_OK)
-			return ret;
+        char *plugin, *plugins_tmp;
 
-		plugin = strtok(NULL, ";");
-	}
+        plugins_tmp = calloc(1, strlen(plugins));
+        if (!plugins_tmp)
+                return VACCEL_ENOMEM;
 
-	return VACCEL_OK;
+        strcpy(plugins_tmp, plugins);
+        plugin = strtok(plugins_tmp, ":");
```

Comment:
I think this causes a memory leak.

We allocate `plugins_tmp` (with `calloc` or `strdup`). `strtok` will forward the pointer in every call, so we loose the initial address of the allocation, which we cannot recover to free. I think we should do something like that:

```
char *plugins_tmp = strdup(plugins);
char *p = plugins_tmp;

plugin = strtok(p, ":");
while (plugin) {
...
}

// Here `plugins_tmp` points to the allocated memory
free(plugins_tmp);
...